### PR TITLE
REST_API_vo_info

### PR DESCRIPTION
**Pull request by ioigoume**
REST API get [vo groups information](https://docs.egi.eu/documentation/498/users/aai/check-in/vos/#api-v2)

### CODE REVIEW

File: content/en/users/aai/check-in/vos/_index.md [hunk 1]
```diff
@@ -299,14 +299,16 @@ expires. Please note that a user with expired membership is not eligible for VO
...
 
 ### API v2
 
-Check-in provides a REST API that allows clients to manage membership
-information only for the VOs they are authoritative for.
+Check-in provides a REST API that allows clients to:
 
-Features:
+- manage membership information only for the VOs they are authoritative for
+- GET VO group information only for the VOs they are authoritative for
```
Comment (gwarf) [inline content/en/users/aai/check-in/vos/_index.md:309]:
```suggestion
- Get VO group information only for the VOs they are authoritative for
```

File: content/en/users/aai/check-in/vos/_index.md [hunk 2]
```diff
@@ -374,7 +433,172 @@ over HTTPS using the Basic Authentication scheme. More sophisticated
...
+
+<!--
+// jscpd:ignore-end
+-->
+
+Response Format:
+
+| HTTP Status        | Description             | Response Body |
+| ------------------ | ----------------------- | ------------- |
+| 200 OK             | Role Returned           | json response |
```
Comment (gwarf) [inline content/en/users/aai/check-in/vos/_index.md:492]:
```suggestion
| 200 OK             | Role Returned           | JSON response |
```

File: content/en/users/aai/check-in/vos/_index.md [hunk 3]
```diff
@@ -374,7 +433,172 @@ over HTTPS using the Basic Authentication scheme. More sophisticated
...
+
+<!--
+// jscpd:ignore-start
+-->
+
+Response Format:
+
+| HTTP Status        | Description             | Response Body |
+| ------------------ | ----------------------- | ------------- |
+| 200 OK             | Role Returned           | json response |
```
Comment (gwarf) [inline content/en/users/aai/check-in/vos/_index.md:537]:
```suggestion
| 200 OK             | Role Returned           | JSON response |
```

File: content/en/users/aai/check-in/vos/_index.md [hunk 4]
```diff
@@ -374,7 +433,172 @@ over HTTPS using the Basic Authentication scheme. More sophisticated
...
```
Comment (gwarf) [inline content/en/users/aai/check-in/vos/_index.md:591]:
```suggestion
| 200 OK             | Role Returned           | JSON response |
```

File: content/en/users/aai/check-in/vos/_index.md [hunk 5]
```diff
@@ -299,14 +299,16 @@ expires. Please note that a user with expired membership is not eligible for VO
...
 
 ### API v2
 
-Check-in provides a REST API that allows clients to manage membership
-information only for the VOs they are authoritative for.
+Check-in provides a REST API that allows clients to:
 
-Features:
+- manage membership information only for the VOs they are authoritative for
+- Get VO group information only for the VOs they are authoritative for
```
Comment (gwarf) [inline content/en/users/aai/check-in/vos/_index.md:309]:
```suggestion
- get VO group information only for the VOs they are authoritative for
```

**Review by gwarf — APPROVED** [review]
LGTM

**Review by vardizzo-lab — APPROVED** [review]
The proposed changes are approved.

**Pull request merged**